packages/run-protocol/src/econ-behaviors.js

```diff
@@ -1,13 +1,20 @@
 // @ts-check
 
 import { E, Far } from '@endo/far';
-import { makeRatio } from '@agoric/zoe/src/contractSupport/index.js';
+import {
+  makeRatio,
+  makeRatioFromAmounts,
+} from '@agoric/zoe/src/contractSupport/index.js';
 import {
   makeGovernedInvitation,
   CONTRACT_ELECTORATE,
   makeGovernedNat,
 } from '@agoric/governance';
-import { CENTRAL_ISSUER_NAME } from '@agoric/vats/src/core/utils.js';
+import {
+  CENTRAL_ISSUER_NAME,
+  ORACLE_PRICE_BRAND_NAME,
+  makeUnitAmount,
```

Comment:
I wonder if `makeUnitAmount` belongs closer to ERTP... it looks like it belongs in the list with `makeRatioFromAmounts` above from `zoe/src/contractSupport`. Or... if it's not used outside of `run-protocol`, perhaps somewhere in `run-protocol`? I have started a `run-protocol/src/contractSupport.js` file in my runStake PR -- stuff that I'm factoring out of vaultFactory.

Reply:
I put it in `zoe/src/contractSupport`.